examples/serve/https/service.yaml

```
@@ -0,0 +1,24 @@
+# SkyServe YAML to run an HTTPS server.
+#
+# Usage:
+# Generate a self-signed certificate and key, or bring your own:
+#   $ openssl req -x509 -newkey rsa:2048 -days 36500 -nodes \
+#       -keyout <key-path> -out <cert-path>
+# Then replace the <key-path> and <cert-path> in the YAML file, and:
+#   $ sky serve up -n https examples/serve/https/service.yaml
+# The endpoint will be printed in the console. You could also
+# check the endpoint by running:
+#   $ sky serve status --endpoint https
+
+service:
+  readiness_probe: /
+  replicas: 1
+  tls:
+    keyfile: <key-path>
+    certfile: <cert-path>
```

Can we make this env var? So we can do `sky serve up --env KEY_FILE --env CERT_FILE`...? And in this file, we make it 


```suggestion
    keyfile: $KEY_FILE
    certfile: $CERT_FILE
 
envs:
  KEY_FILE:
  CERT_FILE:
```